EpiAware/src/EpiAwareUtils/prefix_submodel.jl

```diff
@@ -17,9 +17,11 @@ Generate a submodel with an optional prefix. A lightweight wrapper around the `@
 using EpiAware, DynamicPPL
 
 submodel = prefix_submodel(FixedIntercept(0.1), generate_latent, string(1), 2)
-submodel
```

Comment:
I wonder if we really gain anything from this doc test now and should instead make it an @example?